### ISSUE TITLE
Small mistake...

### ISSUE THREAD

**Issue by hbrecht** [czjvic/php-7.1-for-netbeans-8.2#1]
Thanks for this readme :+1: 
There is a small mistake in #8

Copy <netbeans_8.2_backup_dir>/php/modules**/org-netbeans-modules-spellchecker-twig.jar** to 
<netbeans_8.2>/php/modules/org-netbeans-modules-twig-php.jar (allow overwrite of the file)

It needs to be the same file name as for the target file. Most likely, people will realize this immediatly, but ...

**Comment by snowjog** [comment]
Yes, both source and destination files for #8 should be /php/modules/org-netbeans-modules-php-twig.jar
